src/main/kotlin/org/rust/ide/template/postfix/RsPostfixTemplateProvider.kt

```diff
@@ -102,6 +102,9 @@ class LetPostfixTemplate(provider: RsPostfixTemplateProvider) :
     PostfixTemplateWithExpressionSelector(null, "let", "let name = expr;", RsExprParentsSelector(), provider) {
     override fun expandForChooseExpression(expression: PsiElement, editor: Editor) {
         if (expression !is RsExpr) return
-        extractExpression(editor, expression, postfixLet = true)
+        extractExpression(
+            editor, expression, postfixLet = true, "Postfix `let` template",
+            "template.postfix.let"
```

Comment:
It would be great not to introduce new hardcoded strings. Especially if take into account #9339

Reply:
Don't worry, I'll fix it later, once #9339 gets a green light. I have a ton of fixes to localizable strings, but I want to minimize merge conflicts in the RsBundle.properties.